pkg/monitors/kubernetes/cluster/metrics/jobs.go

```diff
@@ -18,38 +18,53 @@ func datapointsForJob(job *batchv1.Job) []*datapoint.Datapoint {
 		"kubernetes_name":      job.Name,
 	}
 
-	return []*datapoint.Datapoint{
-		datapoint.New(
-			"kubernetes.job.completions",
-			dimensions,
-			datapoint.NewIntValue(int64(*job.Spec.Completions)),
-			datapoint.Gauge,
-			time.Now()),
```

Comment:
Take out all of the `time.Now()`s and replace them with `time.Time{}`.  You are making a bunch of system calls every time you make a datapoint and you don't really need a timestamp anyway since higher up code in this monitor will set the timestamp later.

Reply:
Should we just change all of these in a separate PR? These are littered all throughout the `kubernetes-monitor`?

Reply:
You can if you want but let's do it in this same release as this.

Reply:
I can rebase after https://github.com/signalfx/signalfx-agent/pull/1156